src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -174,35 +193,131 @@
       }
     }
 
-    &-invocation-content {
-      flex: 0 1 27%;
+    &-invocation-header {
       display: flex;
+      flex: 1;
+      justify-content: end;
+      align-items: baseline;
+      font-size: 14px;
+      font-weight: 500;
+      color: $primary;
+      height: 40px;
+
+      &_drift-icon-contrainer {
+        align-self: stretch;
+        margin-right: 2px;
+
+        svg {
+          box-sizing: content-box;
+          padding-top: 8px;
+          width: 17px;
+          height: 17px;
+        }
+      }
+
+      &_drift_up {
+        color: $brightTurquoise;
+      }
+
+      &_drift_down {
+        color: $ceriseRed;
+      }
+
```

Comment:
you're still using single underscore when it can't be used. Single underscore is for modifiers only. In the code above the underscore can be used only for `_up` and `_down`

Reply:
done